### PR TITLE
customize email templates by selection plan

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=2875188&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

File: app/Models/Foundation/Summit/SelectionPlan.php [hunk 1]
```diff
@@ -643,4 +661,51 @@ public function setSubmissionPeriodDisclaimer(?string $submission_period_disclai
...
+     */
+    public function setPresentationCreatorNotificationEmailTemplate(?string $presentation_creator_notification_email_template): void
+    {
+        $this->presentation_creator_notification_email_template = $presentation_creator_notification_email_template;
+    }
+
+    /*
+     * @return String | null
+     */
+    public function getPresentationModeratorNotificationEmailTemplate(): string
```
Comment (smarcet) [inline app/Models/Foundation/Summit/SelectionPlan.php:683]: this return needs to be ?string

File: app/Models/Foundation/Summit/SelectionPlan.php [hunk 2]
```diff
@@ -643,4 +661,51 @@ public function setSubmissionPeriodDisclaimer(?string $submission_period_disclai
         $this->submission_period_disclaimer = $submission_period_disclaimer;
     }
 
+    /*
+     * @return String | null
+     */
+    public function getPresentationCreatorNotificationEmailTemplate(): string
```
Comment (smarcet) [inline app/Models/Foundation/Summit/SelectionPlan.php:667]: this return need to be ?string

File: app/Models/Foundation/Summit/SelectionPlan.php [hunk 3]
```diff
@@ -643,4 +661,51 @@ public function setSubmissionPeriodDisclaimer(?string $submission_period_disclai
...
+     */
+    public function setPresentationModeratorNotificationEmailTemplate(?string $presentation_moderator_notification_email_template): void
+    {
+        $this->presentation_moderator_notification_email_template = $presentation_moderator_notification_email_template;
+    }
+
+    /*
+     * @return String | null
+     */
+    public function getPresentationSpeakerNotificationEmailTemplate(): string
```
Comment (smarcet) [inline app/Models/Foundation/Summit/SelectionPlan.php:699]: this return needs to be ?string

File: app/Models/Foundation/Summit/SelectionPlan.php (hunk 2 above)
Comment (romanetar) [reply app/Models/Foundation/Summit/SelectionPlan.php:667]: resolved in the migration as discussed

File: app/Models/Foundation/Summit/SelectionPlan.php (hunk 1 above)
Comment (romanetar) [reply app/Models/Foundation/Summit/SelectionPlan.php:683]: resolved in the migration as discussed

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**